container/src/main.rs

```diff
@@ -0,0 +1,64 @@
+#![feature(try_from)]
+extern crate clap;
+extern crate holochain_container_api;
+extern crate holochain_core_types;
+
+use clap::{App, Arg};
+use holochain_container_api::{
+    config::{load_configuration, Configuration},
+    container::Container,
+};
+use holochain_core_types::error::HolochainError;
+use std::{convert::TryFrom, fs::File, io::prelude::*};
+
+fn main() {
+    let matches = App::new("hcc")
```

Comment:
Please use `StructOpt` for the Cli. It avoids duplicating a lot of code and writing a lot of boilerplate

Reply:
thanks 🙏